lib/ip_blocklist.js

```diff
@@ -146,6 +146,10 @@ module.exports = function (log, config) {
 
         return result
       })
+
+      return {
+        found: found
+      }
```

Comment:
Yea, it was to keep symmetry since they have similar interface and made testing much easier. With change from above, I might not need this anymore.